content/guide/setup.md

```diff
@@ -22,36 +21,51 @@ Now you’re ready to install and run Quire. And don’t worry, you only have to
 Open your your Terminal command line application and one-by-one copy and paste the following commands. First to install support software for Quire, and then Quire itself:
 
 
-1.) Install Apple’s Xcode if you don’t already have it:
+1.) Check if you have Apple's Xcode 9 installed by copying and pasting the following command:
+
+```
+which xcode-select
+```
+
+And install **Apple’s Xcode** if you don’t already have it:
 
 ```
 xcode-select --install
 ```
 
-2.) Install Homebrew, which helps manage other installs:
+2.) Install **[Homebrew](https://brew.sh)**, which helps manage other installs:
 
 ```
 /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
 ```
 
-3.) Install Node:
+As shown in the screenshots, you’ll need to press return/enter when prompted and your computer password when asked:
+
+![Press Enter](images/homebrew_press-enter.tif)
```

Comment:
All images should be JPG or PNG, and no larger than 1000px on the longest side

Reply:
OK

Reply:
One of the images is a GIF but I don't know if it works... (Here is an example of someone using a gif on Github: https://github.com/fphilipe/PHFComposeBarView)